cdc/processor/sourcemanager/engine/pebble/encoding/key.go

```diff
@@ -96,9 +103,21 @@ func EncodeKey(uniqueID uint32, tableID uint64, event *model.PolymorphicEvent) [
 	// startTs
 	binary.BigEndian.PutUint64(uint64Buf[:], event.StartTs)
 	buf = append(buf, uint64Buf[:]...)
-	// Let Delete < Put
-	binary.BigEndian.PutUint16(uint64Buf[:], ^uint16(event.RawKV.OpType))
+	// Let Delete < Update < Insert
+	binary.BigEndian.PutUint16(uint64Buf[:], getDMLOrder(event.RawKV))
 	buf = append(buf, uint64Buf[:2]...)
 	// key
 	return append(buf, event.RawKV.Key...)
 }
+
+// getDMLOrder returns the order of the dml types: delete<update<insert
+func getDMLOrder(rowKV *model.RawKVEntry) uint16 {
+	// delete
+	if rowKV.OpType == model.OpTypeDelete {
+		return typeDelete
```

Comment:
How about moving the comment `// delete` to here?

Reply:
Or we can just remove comments here. The code itself is clear enough.

Reply:
keep comment above update is enough

Reply:
removed